examples/stdlib/bytearray/bytearray_compare.effekt

```diff
@@ -3,12 +3,51 @@ module examples/pos/bytearray_compare
 import bytearray
 
 def main() = {
-  val b1 = fromString("Hey")
-  val b2 = fromString("Hey")
-  val b3 = fromString("He")
-  val b4 = fromString("Heys")
-  println(compareByteArray(b1, b1))
-  println(compareByteArray(b1, b2))
-  println(compareByteArray(b1, b3))
-  println(compareByteArray(b1, b4))
+  val empty: ByteArray = "".fromString
```

Comment:
I'm still not 100% happy with the test, but at least all backends always return the same thing, so it's at least consistent :shipit: